### PR TITLE
Update eslint

### PR DESCRIPTION
**Pull request by kassens**
Ran this to confirm we're not seeing new lint warnings:

    npm run lint

### CODE REVIEW

**Comment by spicyj** [comment]
eslint 1 disables all the formerly-default rules. Can we copy from fbjs? Like #4746.

**Comment by kassens** [comment]
Added `fbjs-scripts` as a dependency and use its eslintrc as the base similar to React. I disabled some lint rules that need a bigger cleanup which I might do in a followup.

File: frontend/flash.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ function flash(node: DOMElement, flashColor: string, baseColor: string, duration
   node.style.transition = 'none';
   node.style.backgroundColor = flashColor;
   // force recalc
-  node.offsetTop;
+  node.offsetTop; // eslint-disable-line no-unused-expressions
```
Comment (spicyj) [inline frontend/flash.js]: Can also write `void node.offsetTop;`.

**Comment by spicyj** [comment]
:+1:

**Pull request merged**